observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -290,6 +291,42 @@ include::snippets/technology-preview.adoc[]
 <6> Disables verifying the server's certificate chain and host name. The default is `+false+`.
 <7> ServerName indicates the name of the server requested by the client to support virtual hosting.
 
+[id="file-exporter_{context}"]
+== File Exporter
+
+The File Exporter writes telemetry data to files in persistent storage and supports file operations such as rotation, compression, and writing to multiple files. With this exporter, you can also use a resource attribute to control file naming. The only required setting is `path`, which specifies the destination path for telemetry files in the persistent-volume file system.
+
+:FeatureName: The File Exporter
+include::snippets/technology-preview.adoc[]
+
+.OpenTelemetry Collector custom resource with the enabled File Exporter
+[source,yaml]
+----
+# ...
+  config: |
+    exporters:
+      file:
+        path: /data/metrics.json # <1>
+        rotation: # <2>
+          max_megabytes: 10 # <3>
+          max_days: 3 # <4>
+          max_backups: 3 # <5>
+          localtime: true # <6>
+        format: proto # <7>
+        compression: zstd # <8>
+        flush_interval: 5 # <9>
+# ...
+----
+<1> The file-system path where the data is to be written. No default.
```

Comment:
```suggestion
<1> The file-system path where the data is to be written. There is no default.
```

Reply:
Thank you, Sara. I've updated the three instances of this in this PR.